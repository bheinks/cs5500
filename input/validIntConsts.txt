0		1
+1		-1  -0				
-2147483647
000000002147483647			
+001
999999999
-999999999

