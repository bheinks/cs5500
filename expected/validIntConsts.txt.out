TOKEN: T_INTCONST  LEXEME: 0
TOKEN: T_INTCONST  LEXEME: 1
TOKEN: T_PLUS      LEXEME: +
TOKEN: T_INTCONST  LEXEME: 1
TOKEN: T_MINUS     LEXEME: -
TOKEN: T_INTCONST  LEXEME: 1
TOKEN: T_MINUS     LEXEME: -
TOKEN: T_INTCONST  LEXEME: 0
TOKEN: T_MINUS     LEXEME: -
TOKEN: T_INTCONST  LEXEME: 2147483647
TOKEN: T_INTCONST  LEXEME: 000000002147483647
TOKEN: T_PLUS      LEXEME: +
TOKEN: T_INTCONST  LEXEME: 001
TOKEN: T_INTCONST  LEXEME: 999999999
TOKEN: T_MINUS     LEXEME: -
TOKEN: T_INTCONST  LEXEME: 999999999
